ATeam-FantasyGame/Player.cpp

```diff
@@ -0,0 +1,15 @@
+#include "Player.h"
+// Player Class
+
+//Constructor
+// Sets the Players name
+Player::Player(std::string name)
+{
+	name = name;
+}
+
+// Changes the Players currentLocation* to a newLocation*
+void Player::walkToLocation(Location* newLocation)
```

Comment:
the method name is fine - though please note, standard naming convention is "set...." (e.g. setLocation().
Consider, maybe as a ToDo later - how you might validate the argument (ie newLocation).
If the calling program passes in a garbage value - you will store the garbage value in the currentLocation variable, and then crash when you try to use it later.  Best practice is to identify bad values as soon as you can - helps speed up debugging.
Lots of options to do this including a Try / Catch block.